index.js

```diff
@@ -795,15 +813,13 @@
   //. ['oo!', 'oo?', 'on!', 'on?', 'no!', 'no?', 'nn!', 'nn?']
   //.
   //. > chainRec(
-  //. .   [],
+  //. .   Array,
   //. .   (next, done, x) => Array.isArray(x) ? x.map(next) : [done(x)],
   //. .   [1, [[2, 3], 4], 5]
   //. . )
   //. [1, 2, 3, 4, 5]
   //. ```
-  var chainRec = function(chainRec, f, x) {
-    return ChainRec.getBoundMethod('chainRec', chainRec)(f, x);
-  };
+  var chainRec = ChainRec.staticMethods.chainRec;
 
   //# filter :: (Monad m, Monoid m a) => (a -> Boolean, m a) -> m a
```

Comment:
Interesting idea! Essentially we'd require `reduce` (from Foldable) rather than `chain` (from Chain). This may be a weaker requirement, but it's certainly not *strictly* weaker. Why do you believe Foldable to be easier to satisfy than Chain?

I made the change to confirm that we can do it. I pushed the commit.